execution/src/evm.rs

```diff
@@ -277,21 +278,23 @@ impl<'a, R: ExecutionRpc> Database for ProofDB<'a, R> {
 
         let slot = H256::from_uint(&slot);
 
-        Ok(match self.accounts.get(&address) {
+        match self.accounts.get(&address) {
             Some(account) => match account.slots.get(&slot) {
-                Some(slot) => *slot,
-                None => *self
+                Some(slot) => Ok(*slot),
+                None => self
                     .get_account(address, &[slot])?
                     .slots
                     .get(&slot)
-                    .unwrap(),
+                    .cloned()
```

Comment:
nit: use `copied()` instead, as U256 is Copy

Reply:
Thanks for pointing this out. There are probably quite a few places where we can be better on copy vs clone.